src/napari_threedee/manipulators/render_plane_manipulator.py

```diff
@@ -30,15 +31,20 @@ def _update_transform(self):
         self._backend._on_transformation_changed()
 
     def _initialize_transform(self):
-        self.origin = np.array(self.layer.plane.position)
-        plane_normal = self.layer.plane.normal
-        self.rotation_matrix = rotation_matrix_from_vectors_3d([1, 0, 0], plane_normal)
+        origin_world = self.layer.data_to_world(self.layer.plane.position)
+        self.origin = np.array(origin_world)
+        plane_normal_data = self.layer.plane.normal
+        plane_normal_world = data_to_world_ray(vector=plane_normal_data, layer=self.layer)
```

Comment:
visually it looks like the manipulator isn't normal to the plane, so maybe the issue lies here?
Isn't the plane.normal relative to the plane.position? While in data_to_world_ray zeroes are used?
To be fair I'd expect a bigger discrepancy though.